Shared/sdk/SharedUtil.SysInfo.h

```diff
@@ -18,6 +18,20 @@
 
 namespace SharedUtil
 {
+    struct WMIProcessorInfo
+    {
+        uint32_t MaxClockSpeed = 0;
+        std::string Name = {};
+        uint32_t NumberOfCores = 0;
+        uint32_t NumberOfLogicalProcessors = 0;
```

Comment:
Indentation is inconsistent.
```suggestion
        uint32_t    MaxClockSpeed = 0;
        std::string Name = {};
        uint32_t    NumberOfCores = 0;
        uint32_t    NumberOfLogicalProcessors = 0;
```